src/tests/ftest/nvme/nvme_pool_exclude.py

```diff
@@ -0,0 +1,252 @@
+                                               '/run/ior/iorflags/*')
+        self.ior_dfs_oclass = self.params.get(
+            "obj_class", '/run/ior/iorflags/*')
+        # Recreate the client hostfile without slots defined
+        self.hostfile_clients = write_host_file(
+            self.hostlist_clients, self.workdir, None)
+        self.pool = None
+        self.out_queue = queue.Queue()
+        self.container_info = {}
+
+    @fail_on(CommandFailure)
+    def get_rebuild_status(self):
+        """Get the rebuild status.
+
+        Returns:
+            str: reuild status
+
+        """
+        data = self.dmg_command.pool_query(self.pool.uuid)
+        return data["rebuild"]["status"]
+
+    @fail_on(CommandFailure)
+    def get_pool_version(self):
+        """Get the pool version.
+
+        Returns:
+            int: pool_version_value
+
+        """
+        data = self.dmg_command.pool_query(self.pool.uuid)
+        return int(data["version"])
+
+    def ior_thread(self, pool, oclass, api, test, flags, results):
+        """Start threads and wait until all threads are finished.
+        Args:
+            pool (object): pool handle
+            oclass (str): IOR object class
+            API (str): IOR API
+            test (list): IOR test sequence
+            flags (str): IOR flags
+            results (queue): queue for returning thread results
+        """
+        processes = self.params.get("slots", "/run/ior/clientslots/*")
+        mpio_util = MpioUtils()
+        if mpio_util.mpich_installed(self.hostlist_clients) is False:
+            self.fail("Exiting Test: Mpich not installed")
+        self.pool = pool
+        # Define the arguments for the ior_runner_thread method
+        ior_cmd = IorCommand()
+        ior_cmd.get_params(self)
+        ior_cmd.set_daos_params(self.server_group, self.pool)
+        ior_cmd.dfs_oclass.update(oclass)
+        ior_cmd.api.update(api)
+        ior_cmd.transfer_size.update(test[0])
+        ior_cmd.block_size.update(test[1])
+        ior_cmd.flags.update(flags)
+        if "-w" in flags:
+            self.container_info["{}{}{}"
+                                .format(oclass,
+                                        api,
+                                        test[0])] = str(uuid.uuid4())
+
+        # Define the job manager for the IOR command
+        manager = Mpirun(ior_cmd, mpitype="mpich")
+        key = "".join([oclass, api, str(test[0])])
+        manager.job.dfs_cont.update(self.container_info[key])
+        env = ior_cmd.get_default_env(str(manager))
+        manager.assign_hosts(self.hostlist_clients, self.workdir, None)
+        manager.assign_processes(processes)
+        manager.assign_environment(env, True)
+
+        # run IOR Command
+        try:
+            manager.run()
+        except CommandFailure as _error:
+            results.put("FAIL")
+
+    def run_ior_thread(self, action, oclass, api, test):
+        """[summary]
```

Comment:
The summary here is missing.

Reply:
Same as above. Can I do it later when the actual test is enabled?

Reply:
@rpadma2 , let's fix this. PLease fill in the summary and repush PR. I'm ok with skipping build and test in your re-push since this test is skipped anyways and it has already gone through a pass of CI in current patch.
I think we should fix documentation now in case a different person enables this test in future.

Reply:
Sure... I can do it...